Unleash.podspec

```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
+  s.name = 'Unleash'
+  s.version = '1.0.0'
+  s.license = 'MIT'
+  s.summary = 'Feature Flagging for Swift'
+  s.homepage = 'https://github.com/silvercar/unleash-client-swift'
+  s.authors = { 'Silvercar Inc.' => 'mobiledevelopers@silvercar.com' }
+  s.source = { :git => 'https://github.com/silvercar/unleash-client-swift.git', :tag => s.version }
+  s.documentation_url = 'https://github.com/silvercar/unleash-client-swift/blob/master/README.md'
+
+  s.ios.deployment_target = '10.0'
+
+  s.swift_versions = ['4.2', '5.0', '5.1']
+
+  s.source_files = 'unleash/*.swift', 'unleash/Data/*.swift', 'unleash/Extensions/*.swift', 'unleash/Models/*.swift', 'unleash/Services/*.swift', 'unleash/Strategies/*.swift', 'unleash/Utilities/*.swift'
```

Comment:
@ericn441 I think you can specify `'unleash/**/*.swift'` if you want to include all swift files found in the root, and sub-directories recursively, of the project. That way, you don't have to be as granular with each child folder.